server/backend/users/views.py

```
@@ -1,27 +1,23 @@
+from django.utils.html import strip_tags
 from django.core.mail import send_mail
-from django.utils.encoding import force_str, DjangoUnicodeDecodeError
-from django.contrib.auth.tokens import default_token_generator
-from django.contrib.sites.shortcuts import get_current_site
+from django.utils.encoding import force_str, DjangoUnicodeDecodeError, force_bytes
 from django.template.loader import render_to_string
-from django.utils.html import strip_tags
-from rest_framework import status
-from rest_framework.permissions import AllowAny
-from decouple import config
 from django.contrib.auth.tokens import default_token_generator
```

Ruff (0.8.2)

Redefinition of unused `default_token_generator` from line 9

Remove definition: `default_token_generator`

(F811)